nautobot/utilities/testing/__init__.py

```diff
@@ -1,3 +1,54 @@
+import time
+
+from celery.contrib.testing.worker import start_worker
+from django.test import tag, TransactionTestCase as _TransactionTestCase
+
+from nautobot.core.celery import app
+
 from .api import *
 from .utils import *
 from .views import *
+
+
+@tag("unit")
+class TransactionTestCase(_TransactionTestCase):
+    """
+    Base test case class using the TransactionTestCase for unit testing
+    """
+
```

Comment:
Can we not just apply `@tag("unit")` directly to `CeleryTestCase` and avoid the need for this intermediary class?

Reply:
We could do that, but it was my suggestion to prepare for future use-cases of importing directly from our test utilities and discouraging from using Django base utilities so that we can more easily inherit the tags without having to worry about it.